core/services/relay/evm/mercury_provider.go

```diff
@@ -0,0 +1,68 @@
+package evm
+
+import (
+	"context"
+
+	"go.uber.org/multierr"
+
+	relaymercury "github.com/smartcontractkit/chainlink-relay/pkg/reportingplugins/mercury"
+	relaytypes "github.com/smartcontractkit/chainlink-relay/pkg/types"
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2/types"
+
+	"github.com/smartcontractkit/chainlink/core/services"
+	"github.com/smartcontractkit/chainlink/core/services/relay/evm/mercury"
+)
+
+var _ relaytypes.MercuryProvider = (*mercuryProvider)(nil)
+
+type mercuryProvider struct {
+	configWatcher *configWatcher
+	transmitter   mercury.Transmitter
+	reportCodec   relaymercury.ReportCodec
+
+	ms services.MultiStart
+}
+
+func (p *mercuryProvider) Start(ctx context.Context) error {
+	return p.ms.Start(ctx, p.configWatcher, p.transmitter)
+}
+
+func (p *mercuryProvider) Close() error {
+	return p.ms.Close()
+}
+
+func (p *mercuryProvider) Ready() error {
+	return multierr.Combine(p.configWatcher.Ready(), p.transmitter.Ready())
+}
+
+func (p *mercuryProvider) Healthy() error {
+	return multierr.Combine(p.configWatcher.Healthy(), p.transmitter.Healthy())
+}
+
+func (p *mercuryProvider) Name() string {
+	return "EVM.MercuryProvider"
```

Comment:
IIUC this should never be hard coded
cc @essamhassan should mercuryProvider have a Logger, even if just for this method?

Reply:
Happy to make this change, out of curiosity why does it need to come from the logger?